ecosystem.jenkins.config.js

```diff
@@ -31,14 +31,42 @@ module.exports = {
 			env: {
 				PORT: '9901',
 				SERVICE_BROKER: 'redis://localhost:6379/0',
-				SERVICE_GATEWAY_VOLATILE: 'redis://localhost:6379/3',
-				STRICT_READINESS_CHECK: false,
-				WS_RATE_LIMIT_ENABLE: false,
+				SERVICE_GATEWAY_REDIS_VOLATILE: 'redis://localhost:6379/3',
+				ENABLE_HTTP_API: 'http-status,http-version3,http-exports',
+				ENABLE_WS_API: 'blockchain,rpc-v3',
+				GATEWAY_DEPENDENCIES: 'indexer,connector',
+				WS_RATE_LIMIT_ENABLE: 'false',
```

Comment:
could we imply a rate limit here since we already have rate limit for http calls?

Reply:
Do you meant adding the following params?
- `WS_RATE_LIMIT_CONNECTIONS`
- `WS_RATE_LIMIT_DURATION`

Reply:
@priojeetpriyom Usually all the deployments are deployed behind a reverse proxy which manages the rate limiting. These options are made available for user's convenience. As mentioned earlier, any improvement necessary can be discussed and addressed in a separate issue. This PR is not meant for such changes.

@nagdahimanshu could you please include the above params in the config files?